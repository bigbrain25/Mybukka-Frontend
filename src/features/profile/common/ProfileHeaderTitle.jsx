import React from 'react';

import PropTypes from 'prop-types';

import Container from 'Components/container';

const ProfileHeaderTitle = ({ firstName, lastName }) => (
  <Container classNames="profile-header">
    <h1 className="profile-header-title">
      {firstName} {lastName}
    </h1>
  </Container>
);

export default ProfileHeaderTitle;

ProfileHeaderTitle.propTypes = {
  firstName: PropTypes.string.isRequired,
  lastName: PropTypes.string.isRequired,
};
