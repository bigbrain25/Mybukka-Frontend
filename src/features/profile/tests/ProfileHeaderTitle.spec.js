import React from 'react';

import ProfileHeaderTitle from '../common/ProfileHeaderTitle';

describe('ProfileHeaderTitle component', () => {
  const props = {
    firstName: 'First',
    lastName: 'Last'
  };

  const wrapper = mount(
    <ProfileHeaderTitle {...props} />
  );

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
