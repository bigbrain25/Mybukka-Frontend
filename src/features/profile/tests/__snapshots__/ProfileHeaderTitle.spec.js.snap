// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileHeaderTitle component renders properly 1`] = `
ReactWrapper {
  Symbol(enzyme.__unrendered__): <ProfileHeaderTitle
    firstName="First"
    lastName="Last"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "firstName": "First",
      "lastName": "Last",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <h1
          className="profile-header-title"
        >
          First
           
          Last
        </h1>,
        "classNames": "profile-header",
      },
      "ref": null,
      "rendered": Object {
        "instance": <div
          class="container profile-header"
        >
          <h1
            class="profile-header-title"
          >
            First
             
            Last
          </h1>
        </div>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="profile-header-title"
          >
            First
             
            Last
          </h1>,
          "className": "container profile-header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": <h1
              class="profile-header-title"
            >
              First
               
              Last
            </h1>,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "First",
                " ",
                "Last",
              ],
              "className": "profile-header-title",
            },
            "ref": null,
            "rendered": Array [
              "First",
              " ",
              "Last",
            ],
            "type": "h1",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "firstName": "First",
        "lastName": "Last",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <h1
            className="profile-header-title"
          >
            First
             
            Last
          </h1>,
          "classNames": "profile-header",
        },
        "ref": null,
        "rendered": Object {
          "instance": <div
            class="container profile-header"
          >
            <h1
              class="profile-header-title"
            >
              First
               
              Last
            </h1>
          </div>,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="profile-header-title"
            >
              First
               
              Last
            </h1>,
            "className": "container profile-header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": <h1
                class="profile-header-title"
              >
                First
                 
                Last
              </h1>,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "First",
                  " ",
                  "Last",
                ],
                "className": "profile-header-title",
              },
              "ref": null,
              "rendered": Array [
                "First",
                " ",
                "Last",
              ],
              "type": "h1",
            },
          ],
          "type": "div",
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
